### PR TITLE
Added Polar Flow Sync v2.6.4

### PR DESCRIPTION
**Pull request by craigedmunds**
Added Polar Flow Sync v2.6.4  http://support.polar.com/uk-en/support/flowsync

### CODE REVIEW

File: Casks/polar-flowsync.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'polar-flowsync' do
```
Comment (vitorgalvao) [inline Casks/polar-flowsync.rb]: Rename to `flowsync`.

File: Casks/polar-flowsync.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  version '2.6.4'
+  sha256 '6a5c3d76b969cfc48c2adb26149e7911fafc1f1ae0b860f72db107e773b072ba'
+
+  url "https://dngo5v6w7xama.cloudfront.net/connect/download/FlowSync_#{version}.pkg"
```
Comment (vitorgalvao) [inline Casks/polar-flowsync.rb]: Needs a `# dngo5v6w7xama.cloudfront.net was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

File: Casks/polar-flowsync.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  name 'Polar FlowSync Software'
+  homepage 'http://support.polar.com/uk-en/support/flowsync'
+  license :closed
+
+  pkg "FlowSync_#{version}.pkg"
+
+  uninstall pkgutil: "FlowSync_#{version}.pkg",
+            quit:    'fi.polar.FlowSync',
+            delete:  '/Applications/FlowSync.app'
```
Comment (vitorgalvao) [inline Casks/polar-flowsync.rb]: Doesn’t the `pkgutil` take care of that?

Comment (craigedmunds) [inline Casks/polar-flowsync.rb]: It doesn't seem to, no. Took me a while to get the uninstall correct :)

**Comment by craigedmunds** [comment]
Hey vitorgalvao, thanks for reviewing. Have updated based on your comments.

pkgutil doesn't seem to stop the processes, or remove the app, hence the additional quit and delete.

**Pull request merged**